subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java

```diff
@@ -751,7 +751,7 @@ public Test setExcludes(Iterable<String> excludes) {
         return this;
     }
 
-    @Internal
+    @Nested
```

Comment:
`TestFramework` has no annotation. We should either add annotations there, or mark this as `@Internal`.

Reply:
Yeah, I fucked it up. Should be fixed now.